frontend/src/main/scala/bloop/engine/BuildLoader.scala

```diff
@@ -240,14 +234,12 @@ object BuildLoader {
       logger: Logger
   )(
       enableMetals: (Option[AbsolutePath], Option[AbsolutePath]) => T
-  ): Coeval[T] = {
-    Coeval.eval {
```

Comment:
Why drop coeval here?  Was it never really useful?

Reply:
> Was it never really useful?

Yep. There was a coeval but it was run immediately or transfromed into a task.